packages/sanity/src/desk/panes/document/DocumentPaneProvider.tsx

```diff
@@ -152,10 +160,29 @@ export const DocumentPaneProvider = memo((props: DocumentPaneProviderProps) => {
     }
   }, [documentId, presenceStore])
 
+  const inspectors: DocumentInspector[] = useMemo(
+    () => inspectorsResolver({documentId, documentType}, [validationInspector, changesInspector]),
```

Comment:
Is there any reason or benefit to configuring these as default inspectors here instead of adding them to `document.inspectors` in the `deskTool` plugin definition?


```ts
import {changesInspector, validationInspector} from '.path/to/inspectors'

export const deskTool = definePlugin(() => ({
  name: '@sanity/desk-tool',
  document: {
    inspectors: [changesInspector, validationInspector],
  },
  // ...
}))

```

Reply:
Ohh, good catch!

Reply:
It works 👍